www/install.rst

```diff
@@ -79,14 +75,32 @@ popular package managers you can install.
 Macports
 ~~~~~~~~
 
-To install the scipy stack for Python 2.7 with `Macports
+To install the SciPy stack for Python 2.7 with `Macports
 <http://www.macports.org>`_ execute this command in a terminal::
 
     sudo port install py27-numpy py27-scipy py27-matplotlib py27-ipython +notebook py27-pandas py27-sympy py27-nose
 
+Homebrew
+~~~~~~~~
+
+At the time of writing (March 2016), `Homebrew <http://brew.sh/>`_ does not
+have the full SciPy stack available (i.e. you cannot do ``brew install <formula>``
+for everything). However, it is a very popular option (perhaps even more so than
+Macports), and given that active work is being done in this `area <https://github.com/Homebrew/homebrew-python>`_,
+by the `Homebrew group <https://github.com/Homebrew>`_ itself, support for the entire SciPy stack in the future
+seems likely. In the meantime, you can do ``brew install gfortran`` followed ``pip
+install <package>`` for each of the individual components of the stack.
 
 .. _individual-packages:
 
+Windows packages
+----------------
+
+Windows does not have any package manager analogous to that in Linux, so installing
+one of the scientific Python distributions mentioned above is preferred. However, if
+that is not an option, Christoph Gohlke provides `pre-built Windows installers <http://www.lfd.uci.edu/~gohlke/pythonlibs/>`_
+for many Python packages, including all of the core SciPy stack, which work extremely well.
```

Comment:
Well, they technically are Windows specific packages (or wheels to be precise) that you can install.  It only seems fair that if Linux and Mac get their own sections, Windows should get its own too?  Otherwise, I get the impression that these doc pages are more Linux-Mac oriented with Windows included on the side as somewhat of an after thought.